sdk-actors/src/test/java/io/dapr/actors/TestUtils.java

```diff
@@ -47,13 +47,11 @@ public static void assertThrowsDaprException(
   public static <T extends Throwable> void assertThrowsDaprException(
       Class<T> expectedType,
       String expectedErrorCode,
-      String expectedErrorMessage,
```

Comment:
@artur-ciocanu why is this gone?

Reply:
instead of deleting it can we fix it forward in a separate PR?

Reply:
Fixed.